sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -223,14 +223,21 @@ private[spark] object SQLConf {
     defaultValue = Some(200),
     doc = "The default number of partitions to use when shuffling data for joins or aggregations.")
 
-  val CODEGEN_ENABLED = booleanConf("spark.sql.codegen",
+  val TUNGSTEN_ENABLED = booleanConf("spark.sql.tungsten.enabled",
```

Comment:
Should we use a better name for this? `tungsten` will not be easy for user to understand.